lib/src/code_field.dart

```diff
@@ -1,18 +1,18 @@
 import 'dart:async';
```

Comment:
Added redundant types.
We will be pulling changes from BertrandBev, so this file should not be edited without need.
Will not merge this.